packages/editor-ui/src/utils/sortUtils.ts

```diff
@@ -240,6 +246,23 @@ export function sublimeSearch<T extends object>(
 				accu: null | { matched: boolean; outScore: number },
 				{ value, weight }: { value: string; weight: number },
 			) => {
+				// Check for whole word match first
+				if (containsWord(value, filter)) {
```

Comment:
can you please add some tests?

Reply:
have you tried increasing the first word bonus? 
Tried increasing `FIRST_LETTER_BONUS` to 30 and that was enough to get agent to top